### PR TITLE
WIP Android sample

### PR DESCRIPTION
**Pull request by hannesa2**

### CODE REVIEW

File: example/android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.squareup.moshi:moshi:1.9.2'
+    // Required if generating JSR-303 annotations
+    implementation 'javax.validation:validation-api:1.1.0.CR2'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.5.1'
+
+}
+
+// Each configuration is set to the default value
+jsonschema2dataclass {
```
Comment (hannesa2) [inline example/android/app/build.gradle:41]: Is this right ?
@eirnym

File: example/android/build.gradle [hunk 2]
```diff
@@ -0,0 +1,17 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:4.1.2'
+        classpath 'org.jsonschema2dataclass:jsonschema2dataclass-gradle-plugin:latest.integration'
```
Comment (hannesa2) [inline example/android/build.gradle:8]: What is right here ?
@eirnym

Comment (eirnym) [reply example/android/build.gradle:8]: should be `gradle.plugin.org.jsonschema2dataclass:jsonschema2dataclass`

File: example/android/app/build.gradle (hunk 1 above)
Comment (eirnym) [reply example/android/app/build.gradle:41]: I haven't changed it yet, so `2pojo` here for now

**Comment by hannesa2** [comment]
Any clue how to solve this 

image

I don't set it at all !

image

**Comment by hannesa2** [comment]
@eirnym Any clue how to solve this 

image

I don't set it at all !

image

**Comment by eirnym** [comment]
Nice, I'll look on it

**Comment by eirnym** [comment]
With this parameter I run into a problem. I'll do the same hack as in the original repo. I think this is a half measure though and have a tendency to rewrite full plugin from scratch after a quick comparison with other gradle plugins

**Comment by eirnym** [comment]
@hannesa2 please tell if I chosen a correct folder for json location. Please correct it based on [AndroidSourceSet](https://developer.android.com/reference/tools/gradle-api/4.1/com/android/build/api/dsl/AndroidSourceSet) docs

**Comment by eirnym** [comment]
PS:fixed.. it's strange, that it wasn't an issue on a local build

**Pull request merged**